src/app/static/src/js/components/runWorkflow/runWorkflowProgress.vue

```diff
@@ -1,19 +1,206 @@
 <template>
-    <p>Run workflow progress is coming soon</p>
+    <div
+        class="container mt-3"
+        id="workflow-progress-container"
+        v-if="workflowRunSummaries"
+    >
+        <div class="row mb-3">
+            <label for="workflows" class="form-label col">Workflow</label>
+            <div class="col-10 px-0">
+                <v-select
+                    :options="workflowRunSummaries.data"
+                    label="name"
+                    :reduce="(label) => label.key"
+                    name="workflows"
+                    id="v-select"
+                    v-model="selectedWorkflowKey"
+                    placeholder="Search by name..."
+                >
+                    <template #option="{ name, date }">
+                        <div>
+                            {{ name }}
+                            <span style="opacity: 0.5; float: right">
+                                {{ formatDate(date) }}</span
+                            >
+                        </div>
+                    </template>
+                </v-select>
+            </div>
+        </div>
+        <div class="row" v-if="workflowRunStatus" id="workflow-table">
+            <label class="form-label col">Reports</label>
+            <table class="table-bordered col-10">
+                <tr v-for="report in workflowRunStatus.data.reports">
+                    <td v-if="report.status === 'success'">
+                        <a
+                            :href="
+                                reportVersionHref(report.name, report.version)
+                            "
+                            >{{ report.name }}</a
+                        >
+                    </td>
+                    <td v-else-if="report.status === 'error'">
+                        <a
+                            :href="reportLogsHref(report.name)"
```

Comment:
Are only error messages going to the logs page? Or should missing, impossible, etc go there as well?

Reply:
The logs page shows all the logging for any report, whatever its status - as well as showing the status itself.